src/Schema/SQLiteSchemaManager.php

```diff
@@ -463,11 +469,14 @@ protected function selectIndexColumns(string $databaseName, ?string $tableName =
             <<<'SQL'
             SELECT t.name AS table_name,
                    i.name,
-                   i."unique"
+                   i."unique",
+                   c.name AS column_name
               FROM sqlite_master t
               JOIN pragma_index_list(t.name) i
+              JOIN pragma_index_info(i.name) c
              WHERE %s
-          ORDER BY t.name, i.seq
+               AND i.name NOT LIKE 'sqlite_%%'
```

Comment:
Is the double percent intended, or should it be the following?
```suggestion
               AND i.name NOT LIKE 'sqlite_%'
```

Reply:
No, it's not.

This string passed to the sprintf

To have percent sign in the result - it should be escaped with percent
https://www.php.net/manual/en/function.sprintf.php#88218

Reply:
Oh yes, thanks 👍